site/guide/manage-groups.qmd

```diff
@@ -14,27 +14,25 @@ To perform these steps, you must be a group administrator or a superuser.
 
 2. Under **User Permissions** in the sidebar, select **Groups**.
 
-3. Click **Add New Group** to initiate the group creation process:
-   - Provide a name for the new group.
-   - Optionally, specify a description for the group's purpose.
-   - Add users to the group by selecting them from a list or specifying their usernames.
-   - Save the new group configuration.
+3. Click **Create New Group** to initiate the group creation process:
+   - Provide a name and description for the new group, click **Continue**.
+   - Select users from the list to add them to the group.
+   - Click **Create Group** to save.
 
 4. Confirm that the new group is created and appears in the list of available groups, ensuring it is correctly configured for its intended use.
 
 ## See group details
 
 1. [Log in to the ValidMind UI]({{< var vm_ui >}}).
 
-2. Access the "Group Management" section.
+2. Click **Settings**. 
```

Comment:
This also seems to be missing a step: you need to click **Organization** before you can continue with the next step.

Reply:
I am including this in the new pull request @nrichers

Reply:
@nrichers I do not see the need to go to Organization for the group settings, in fact the correct landing page should be  https://app.dev.vm.validmind.ai/settings/groups instead of https://app.dev.vm.validmind.ai/settings/organization

So the below are the corrected steps - can you review:
_____________________

## See group details

1. [Log in to the ValidMind UI]({{< var vm_ui >}}).

2. Click **Settings**.

3. Access the "Groups" section.

4. Locate the group for which you want to see details, expand **See Details** to:
   - View the group's name and description.
   - See a list of users who are members of the group.
   - Review any additional information or settings related to the group.

Reply:
@subatraSR2 yes, you are correct. I compared your steps against the UI and the missing intermediate step takes you to **Groups**. 👍